lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -584,6 +584,7 @@ describe('AwsInvokeLocal', () => {
 
       expect(serverless.cli.consoleLog.lastCall.args[0]).to.contain('"errorMessage": "failed"');
       expect(serverless.cli.consoleLog.lastCall.args[0]).to.contain('"errorType": "Error"');
+      expect(serverless.cli.consoleLog.lastCall.args[0]).to.contain('"stackTrace": [');
```

Comment:
why the `[`?

Reply:
It’s to indicate that `stackTrace` should be an array.